package.json

```diff
@@ -5,7 +5,9 @@
     "node": ">=10"
   },
   "dependencies": {
-    "swagger-ui": "3.24.3"
+    "swagger-ui": "3.24.3",
+    "fomantic-ui": "2.8.3",
+    "gulp": "4.0.2"
```

Comment:
Please remove again.

Reply:
```suggestion
```

Reply:
Also, don't forget to regenerate `package-lock-json`, e.g. `npm i --package-lock-only`.

Reply:
done.